content/people/aalsolim.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Anas Alsoliman
+role: student-PhD
+status: Ph.D. Candidate
+yearjoined: 2017
+portrait: aalsolim.jpg
+badge: aalsolim-badge.jpg
+interests:
+  - Vehicular Networks
+  - Connected-Cars/Autonomous Vehicles networking applications
```

Comment:
Is the dash in “Connected-Cars” necessary? If this is indeed how the
name of that field of research is written, ignore this comment.

Reply:
I did a quick research but I don't know as well. 
I'd keep it for now since she cared enough to write it.

Reply:
All right. Just wanted to make sure it is not a mistake, as it looks
somewhat unusual.